Use find_library instead of dependency so llvm-libunwind can be used,
since it doesn't ship a pkg-config file.

diff --git a/meson.build b/meson.build
index 448e262..1a8ebfb 100644
--- a/meson.build
+++ b/meson.build
@@ -1,11 +1,12 @@
 project(
-    'Tilix', 'd',
+    'Tilix', ['c','d'],
     version: '1.9.4',
     license: 'MPL-2.0',
     meson_version: '>= 0.52'
 )
 
 compiler = meson.get_compiler('d')
+cc = meson.get_compiler('c')
 if compiler.get_id() == 'llvm'
   d_extra_args = ['-vcolumns']
   d_link_args = []
@@ -100,7 +101,7 @@ sources_dir = include_directories('source/')
 gtkd_dep = dependency('gtkd-3', version: '>=3.8.5')
 vted_dep = dependency('vted-3', version: '>=3.8.5')
 xlib_dep = dependency('x11')
-libunwind_dep = dependency('libunwind')
+libunwind_dep = cc.find_library('libunwind')
 libsecret_dep = dependency('libsecret-1', required: false)
 
 subdir('po')
